include/af/moments.h

```diff
@@ -0,0 +1,83 @@
+/*******************************************************
```

Comment:
Make only the functions listed in this file part of `image.h`. We should avoid creating new header files unless the functions are of a completely new domain.